### PR TITLE
GG-18760 Configuration tests moved to separate module. 

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/web-console-configuration-tests/pom.xml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        <artifactId>community.parent</artifactId>
+        <version>1</version>
+        <relativePath>../../../parent</relativePath>
+    </parent>
+
+    <artifactId>ignite-web-console-configuration-tests</artifactId>
+    <version>9.0.0-SNAPSHOT</version>
+    <url>http://ignite.apache.org</url>
+
+    <profiles>
```
Comment (nva) [inline modules/web-console/web-console-configuration-tests/pom.xml]: Remove this profiles

File: modules/web-console/web-console-configuration-tests/pom.xml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                    <exclude>**/*.java</exclude>
+                </excludes>
+            </testResource>
+            <testResource>
+                <directory>src/main/resources</directory>
+            </testResource>
+        </testResources>
+
+        <plugins>
+            <plugin>
```
Comment (nva) [inline modules/web-console/web-console-configuration-tests/pom.xml]: Remove this plugin

File: modules/web-console/web-console-configuration-tests/pom.xml [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                            <excludeTransitive>false</excludeTransitive>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
```
Comment (nva) [inline modules/web-console/web-console-configuration-tests/pom.xml]: Remove this configuration

File: modules/web-console/web-console-configuration-tests/pom.xml (hunk 2 above)
Comment (vsisko) [reply modules/web-console/web-console-configuration-tests/pom.xml]: Removed.

File: modules/web-console/web-console-configuration-tests/pom.xml (hunk 1 above)
Comment (vsisko) [reply modules/web-console/web-console-configuration-tests/pom.xml]: Removed.

File: modules/web-console/web-console-configuration-tests/pom.xml (hunk 3 above)
Comment (vsisko) [reply modules/web-console/web-console-configuration-tests/pom.xml]: Removed.

**Pull request merged**